pkg/app/piped/cloudprovider/kubernetes/diff.go

```diff
@@ -57,6 +57,13 @@ func Diff(old, new Manifest, opts ...diff.Option) (*diff.Result, error) {
 			return nil, err
 		}
 	}
+
+	var err error
+	old.u = remarshal(old.u)
```

Comment:
I think you are adding the error handling, right?
So I will skip reviewing this PR for a while.
Please ping me if your PR is ready to review.

Reply:
Yes, I'm adding error handling. But I'm not sure how to deal [this comment line](https://github.com/pipe-cd/pipecd/pull/3370#discussion_r822481942), so I have not added all yet. I should have mentioned about that. Sorry.

Reply:
@nghialv